src/structures/Guild.js

```diff
@@ -737,14 +749,43 @@ class Guild extends Base {
    */
   fetchVanityCode() {
     if (!this.features.includes('VANITY_URL')) {
-      return Promise.reject(new Error('VANITY_URL'));
+      throw new Error('VANITY_URL');
```

Comment:
As this isn't an async function, if you use throw, you'll get a sync error not an async one
```suggestion
      return Promise.reject(new Error('VANITY_URL'));
```